### ISSUE TITLE
[Anon] Bug report: Chaos Space Marines - Codex.catz

### ISSUE THREAD

**Issue by BSDataAnon** [BSData/wh40k-7th-edition#789]
**File:** Chaos Space Marines - Codex.catz

**BattleScribe version:** 1.15.01

**Platform:** Android

**Dropbox:** No

**Description:** Lots or error for the cultists. The cost for the units, the cost for the autogun. And the cumulative cost doesn't work

**Comment by deepsleeper** [comment]
- Add to this that the cost for the CSM (troops) is wrong. Suppose to be 75 pts 
- As for the cultists, if you add more unit, the total cost dosen't change.

**Comment by NebSeele** [comment]
As mentioned in #777, this bug appears to me that modifier related costs still don't show until it's being added into the roster (creating discrepancies in cost). This is a battlescribe problem that can be annoying, but not unworkable.

I have reworked the cultist entry slightly so that the cultists in total option has the base cost, with autoguns having the 1pt upgrade cost. Ditto for CSM and their options. Both probably should have been done when those options were added to balance out the heavy weapons issues, though being busy IRL and trying to get the BL and CS updated made me miss an obvious choice.

The bug will be fixed in the next update.
